src/components/Table/_table.scss

```diff
@@ -1 +1,23 @@
 @import 'TableHead/table-head';
+@import '~carbon-components/scss/components/data-table/data-table';
+
+.toast {
+  margin-bottom: 0.5rem;
+  min-width: 30rem;
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+}
+
+.buttons {
+  width: 88px;
+  height: 32px;
+  border-color: #0062ff;
+  border-width: 1px;
+  min-height: auto;
+  margin: auto 8px;
```

Comment:
we should be using carbon tokens for spacing and we should not be using px values.